gcp/src/gcpRequest.ts

```diff
@@ -26,8 +26,12 @@ export class GcpRequest implements CloudRequest {
   private parseJson(body: any) {
```

Comment:
Can we change this so it returns the appropiate type? String if it is a string or an Object if its a json object

Reply:
It allows both, string and object, in each case take a different flow